make/config.mk

```diff
@@ -114,3 +114,11 @@ EXTRA_OPERATORS =
 # whether to use torch integration. This requires installing torch.
 USE_TORCH = 0
 TORCH_PATH = $(HOME)/torch
+
+# whether to use sframe integration. This requires build sframe
+# git@github.com:dato-code/SFrame.git
+USE_SFRAME = 0
+SFRAME_PATH = $(HOME)/SFrame
+ifeq ($(USE_SFRAME), 1)
+	MXNET_PLUGINS += plugin/sframe/SFrame.mk
```

Comment:
User can always add sframe by using two lines, keep these lines in comment

SFRAME_PATH=xxx
MXNET_PLUGINS += plugin/sframe/plugin.mk